### PR TITLE
Integrate website choice screen DESKTOP-1663

### PR DESCRIPTION
**Pull request by awendland**
Integrates the screen for choosing which website proof to use (either file, a catch-all term for http/https, or dns).

Minor `console.warn` addition to meta navigator that will report if the provided subRoute does not have a parseRoute method (an error that I didn't recognize for a bit and is easy to protect against).

### CODE REVIEW

**Comment by awendland** [comment]
@keybase/react-hackers

File: shared/router/meta-navigator.js [hunk 1]
```diff
@@ -118,6 +118,9 @@ class MetaNavigator extends Component {
         if (nextPath && subRoutes[nextPath.get('path')]) {
           nextComponent = subRoutes[nextPath.get('path')]
           parseNextRoute = nextComponent.parseRoute
+          if (!parseNextRoute) {
+            console.warn(`MetaNavigator: sub-route '${nextPath.get('path')}' lacks a static parseRoute function`)
```
Comment (MarcoPolo) [inline shared/router/meta-navigator.js]: 👍

File: shared/profile/index.js [hunk 2]
```diff
@@ -40,6 +41,7 @@ class Profile extends Component<void, ?Props, void> {
       subRoutes: {
         'editprofile': EditProfile,
         ProveEnterUsername,
+        ProveWebsiteChoice,
```
Comment (MarcoPolo) [inline shared/profile/index.js]: 🐑 I'd call this ProveWebsiteChoiceContainer to be a little less confusing

**Comment by MarcoPolo** [comment]
👍

**Pull request merged**